etc/templates/compute.glue21

```diff
@@ -364,10 +368,31 @@ GLUE2EntityOtherInfo: ${other}
 % endfor
 ## XXX Missing
 ## GLUE2CloudComputingImageContextualizationVersion:
+## XXX Not exposed for security reasons
 ## GLUE2CloudComputingImageDefaultUsername:
 ## GLUE2CloudComputingImageDefaultPassword:
-## GLUE2CloudComputingImageVersion:
 
+## All NetworkTraffic for $img_id
```

Comment:
would be good to have some tests on the output of the templates, they keep growing and we do not have good understanding of the output